classes/test.php

```diff
@@ -396,6 +396,16 @@ public function setAssertionManager(test\assertion\manager $assertionManager = n
 			->setPropertyHandler('exception', function() { return asserters\exception::getLastValue(); })
 		;
 
+		// Around testedInstance
+		$this->assertionManager
+			->setHandler('callStaticOnTestedClass', function() use ($test) {
```

Comment:
@mikaelrandy what is not correct is the indentation inside the closure : we always add one extra level for nested closures ;) See the closing brace (`}`)  and the closing paren. (`)`) ?